satpy/readers/mirs_l2_nc.py

```diff
@@ -0,0 +1,506 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2019 Satpy developers
+#
+# This file is part of satpy.
+#
+# satpy is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later
+# version.
+#
+# satpy is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR
+# A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# satpy.  If not, see <http://www.gnu.org/licenses/>.
+"""Interface to MIRS product."""
+
+import os
+import logging
+import datetime
+import numpy as np
+import xarray as xr
+import dask.array as da
+from pyproj import CRS
+from satpy.readers.file_handlers import BaseFileHandler
+
+LOAD_CHUNK_SIZE = int(os.getenv('PYTROLL_LOAD_CHUNK_SIZE', -1))
```

Comment:
This should be taken `from satpy import CHUNK_SIZE` so it can be updated more easily in the future. The environment variable is `PYTROLL_CHUNK_SIZE` that controls that value.